neo/Ledger/Blockchain.cs

```diff
@@ -610,6 +610,25 @@ private void Persist(Block block)
                 foreach (IPersistencePlugin plugin in Plugin.PersistencePlugins)
                     plugin.OnPersist(snapshot, all_application_executed);
                 snapshot.Commit();
+                List<Exception> commitExceptions = null;
+                foreach (IPersistencePlugin plugin in Plugin.PersistencePlugins)
```

Comment:
i think that is better to catch only the first exception, with only one `try`, not inside the foreach.

Reply:
The idea is to let them all run; since the snapshot has already been persisted it takes the least effort to only need to fix up the one that failed commit. Failing commit is a should never happen scenario. So it is best not to stop the others. 

Also, in the future we could make it execute them in parallel. Plugins could be writing to different partitions and would be able to take advantage of running Commit in parallel in different threads.